deploy/docker/Dockerfile

```diff
@@ -1,5 +1,10 @@
-FROM BASEIMAGE
+FROM golang:1.13.7 as builder
+COPY . /go/src/github.com/kubernetes-incubator/metrics-server
+RUN cd /go/src/github.com/kubernetes-incubator/metrics-server && \
+    make _output/amd64/metrics-server
 
-COPY metrics-server /
+FROM gcr.io/distroless/static:latest
```

Comment:
Can you pin to a specific release (sha). Also make sure to use the nonroot variant.

Reply:
Pinned to latest nonroot sha